pkgs/development/tools/electron/common.nix

```diff
@@ -234,6 +234,7 @@ in (chromium.override { upstream-info = info.chromium; }).mkDerivation (base: {
   meta = with lib; {
     description = "Cross platform desktop application shell";
     homepage = "https://github.com/electron/electron";
+    knownVulnerabilities = optional (versionOlder info.version "29") "Electron version ${version} is EOL";
```

Comment:
Don't we usually remove the EOL `electron-source` versions and mark the binary `electron_*-bin` version EOL?
Not both?

See #295672 (c45702dfc951818ae096aafc34b0eea990bc9a36 and 1f9c426744ca908110a8ab392352c843ca07a7c3).